policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -225,14 +230,69 @@ impl kubert::index::IndexNamespacedResource<Service> for Index {
 
     fn delete(&mut self, namespace: String, name: String) {
         tracing::debug!(name, namespace, "deleting service");
-        let service_ref = ServiceRef { name, namespace };
+        let service_ref = ResourceRef { name, namespace };
         self.service_info.remove(&service_ref);
         self.services_by_ip.retain(|_, v| *v != service_ref);
 
         self.reindex_services()
     }
 }
 
+impl kubert::index::IndexNamespacedResource<linkerd_k8s_api::UnmeshedNetwork> for Index {
+    fn apply(&mut self, u: linkerd_k8s_api::UnmeshedNetwork) {
+        let um: UnmeshedNetwork = u.into();
+        tracing::debug!(um.name, um.namespace, "indexing unmeshed network");
+
+        self.unmeshed_networks.insert(
+            ResourceRef {
+                name: um.name.clone(),
+                namespace: um.namespace.clone(),
+            },
+            um,
+        );
+    }
+
+    fn delete(&mut self, namespace: String, name: String) {
+        tracing::debug!(name, namespace, "deleting unmeshed networks");
+        let um_ref = ResourceRef { name, namespace };
+        self.unmeshed_networks.remove(&um_ref);
+    }
+}
+
+impl kubert::index::IndexNamespacedResource<Pod> for Index {
+    fn apply(&mut self, pod: Pod) {
+        let ns = pod.namespace().unwrap();
```

Comment:
nit: we can call this variable `namespace`, and then use shorthand notation in the `ResourceRef` item below.

Reply:
This is fairly idiomatic in this codebase already.
```
:; rg 'let ns =' -c -t rust
policy-controller/src/admission.rs:1
policy-controller/k8s/index/src/outbound/index.rs:4
policy-controller/k8s/index/src/inbound/server_authorization.rs:1
policy-controller/k8s/index/src/inbound/index.rs:13
policy-controller/k8s/index/src/inbound/meshtls_authentication.rs:1
policy-test/src/lib.rs:1
```